### PR TITLE
task-2

### PR DESCRIPTION
**Pull request by Krystsia**
S3Bucket: http://krystsia-second-app.s3-website-us-east-1.amazonaws.com/
Cloudfront: https://d3j6t7dtp285iz.cloudfront.net/

manual  S3Bucket: http://krystsia-first-app.s3-website-us-east-1.amazonaws.com/
manual Cloudfront: https://d2dqm8w6ufyw5l.cloudfront.net/

### CODE REVIEW

**Comment by Shestak2039** [comment]
Can you share a link to manual and automatic S3 and cloudFront?
image

**Comment by Krystsia** [comment]
> 
> 
> Can you share a link to manual and automatic S3 and cloudFront?
> image

I've updated the comment above with links to manual created S3 backet and Cloudfront disctribution

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+#app: your-app-name
+#org: your-org-name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: '2'
+
+provider:
+  name: aws
+  runtime: nodejs12.x
```
Comment (Student-Java) [inline serverless.yml:25]: it's a better idea to set a region explicitly

**Comment by Shestak2039** [comment]
Well done!

**Review by Shestak2039 — APPROVED** [review]

**Pull request merged**